src/components/Heroes/index.jsx

```diff
@@ -30,25 +27,28 @@ class RequestLayer extends React.Component {
       }),
     }),
     strings: PropTypes.shape({}),
-  }
+  };
 
   componentDidMount() {
     this.props.dispatchHeroStats();
     this.props.onGetProPlayers();
   }
 
-  getMatchCountByRank = (json, rank) => json.map(heroStat => heroStat[rank] || 0).reduce(sum, 0) / 10
+  getMatchCountByRank = (json, rank) =>
+    json.map((heroStat) => heroStat[rank] || 0).reduce(sum, 0) / 10;
 
   createTab = (key, matchCount) => {
     const { strings } = this.props;
 
-    const name = key === 'public'
-      ? strings.hero_public_tab
-      : strings.hero_pro_tab;
+    let name;
```

Comment:
Think the object syntax is cleaner:
https://github.com/odota/web/pull/2702/files#diff-e4297e4cfca8a4227e2b4d6be27868b995f9502019f376296ece7f5fba2f4e02R45

Reply:
I had no idea of that approach and of that syntax, it indeed looks cleaner.